### ISSUE TITLE
windows nyc compilation issue

### ISSUE THREAD

**Issue by CuzImClicks** [andrewgazelka/hyperion#500]
`cargo run --bin nyc`
```
error: couldn't read \\?\C:\Users\\Desktop\hyperion\target\debug\build\typenum-101e9472ade95506\out/op.rs: The filename, directory name, or volume label syntax is incorrect.. (os error 123)
  --> C:\Users\\.cargo\registry\src\index.crates.io-6f17d22bba15001f\typenum-1.17.0\src\lib.rs:72:5
   |
72 |     include!(concat!(env!("OUT_DIR"), "/op.rs"));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `typenum` (lib) due to 1 previous error
```

**Comment by andrewgazelka** [comment]
going to fix with #491

**Comment by andrewgazelka** [comment]
use docker
